data/json/items/comestibles/med.json

```diff
@@ -333,8 +331,6 @@
         "weight" : 240,
         "volume" : 1,
         "price" : 3000,
-        "charges" : 1,
-        "stack_size" : 1,
```

Comment:
But isn't the ability to stack one by itself and not being able to stack any the same thing?